docs/proposals/scheduling/20220722-multi-hierarchy-elastic-quota-management.md

```diff
@@ -331,6 +331,7 @@ we will also add new annotation and labels to achieve our desired functionality.
 annotations:
   quota.scheduling.koordinator.sh/runtime: {cpu:4, memory: 8Gi}
   quota.scheduling.koordinator.sh/shared-weight: {cpu:4, memory: 8Gi}
+  quota.scheduling.koordinator.sh/namespaces: "namespace1,namespace2"
```

Comment:
We should keep the same style as other annotations, so JSON arrays are recommended for namespaced values.

Reply:
fixed

Reply:
fixed